scripts/dataset_processing/g2p/export_zh_cpp_data_to_manifest.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import json
+from argparse import ArgumentParser
+from collections import defaultdict
+from glob import glob
+from typing import List, Tuple
+
+from tqdm import tqdm
+
+"""
+Converts CPP data to .json manifest format for Chinese HeteronymClassificationModel training.
+Chinese dataset could be found here:
+    https://github.com/kakaobrain/g2pM#the-cpp-dataset
+
```

Comment:
Could you add here steps on how to run this script from the PR description? `git clone ...`
It would be great if you could also update the docs here https://github.com/NVIDIA/NeMo/blob/main/docs/source/tts/g2p.rst